packages/core/schematics/utils/import_manager.ts

```diff
@@ -29,6 +29,13 @@ export class ImportManager {
       new Map<ts.ImportDeclaration, {propertyName?: ts.Identifier, importName: ts.Identifier}[]>();
   /** Map of source-files and their previously used identifier names. */
   private usedIdentifierNames = new Map<ts.SourceFile, string[]>();
+  /** Map of source files and the new imports that have to be added to them. */
+  private newImports: Map<ts.SourceFile, {
+    importStartIndex: number,
+    defaultImports: Map<string, ts.Identifier[]>,
```

Comment:
Do you recall in what cases the default import for a module needs multiple identifiers? Aren't we already deduping/using the cached identifiers. Maybe should be just a single `ts.Identifier`?

```suggestion
    defaultImports: Map<string, ts.Identifier>,
```

Reply:
I don't remember exactly why it was written this way and we don't have any using it so it's hard to tell so I kept the default import logic as it was. My best guess is that it might not be safe to reuse a default import to a file, because it can be referenced under different names in the same file.

Reply:
I think we would never hit the case where multiple identifiers would be pushed to the array:

see: https://github.com/angular/angular/blob/f024ce44512ce92416f7af7a7008b02a2bdcd6cc/packages/core/schematics/utils/import_manager.ts#L67

Reply:
I see. Changed.